src/plugins/console/public/application/components/help_panel.tsx

```diff
@@ -43,9 +44,30 @@ export function HelpPanel(props: Props) {
           <p>
             <FormattedMessage
               id="console.helpPage.requestFormatDescription"
-              defaultMessage="You can type one or more requests in the white editor. Console understands requests in a compact format:"
+              defaultMessage="You can type one or more requests in the editor. Console understands requests in a compact format."
             />
           </p>
+          <p>
+            <FormattedMessage id="console.helpPage.learnAbout" defaultMessage="Learn about" />
+            &nbsp;
+            <EuiLink
+              href="https://www.elastic.co/guide/en/kibana/current/console-kibana.html"
+              target="_blank"
+              external
+            >
+              Console
```

Comment:
should be localized

Reply:
I think it's OK not to localize this, since it's a product name.

Reply:
@debadair Can you confirm?